src/helpers/figures/charts/scorecard_chart_config_builder.ts

```diff
@@ -269,37 +353,27 @@ class ScorecardChartConfigBuilder {
           this.secondaryFontColor,
       },
       baselineDescr: {
-        font: getDefaultContextFont(baselineFontSize * BASELINE_DESCR_FONT_RATIO),
+        font: getDefaultContextFont(baselineDescrFontSize),
+        splitted: splittedBaseline,
         color: this.secondaryFontColor,
       },
       baselineArrow:
-        this.baselineArrow === "neutral"
+        this.baselineArrow === "neutral" || this.runtime.progressBar
           ? undefined
           : {
-              size: this.keyValue ? 0.8 * baselineFontSize : 0,
+              size: this.keyValue ? 0.8 * baselineValueFontSize : 0,
               color: this.runtime.baselineColor || this.secondaryFontColor,
             },
     };
   }
 
   /** Get the height of the chart minus all the vertical paddings */
-  private getDrawableHeight(): number {
-    const verticalPadding = 2 * this.chartPadding;
-    let availableHeight = this.height - verticalPadding;
+  private get getDrawableHeight(): number {
```

Comment:
Why this change ?
Either it's a getter and the name should not start with `get` or it's a function.
```suggestion
  private getDrawableHeight(): number {
```